dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -839,7 +839,8 @@ private synchronized Host createSystemHost() throws DotDataException,
             this.systemHost = systemHost;
         } else {
             final String systemHostId = (String) rs.get(0).get("id");
-            this.systemHost =  APILocator.getHostAPI().DBSearch(systemHostId, systemUser, false);
+            this.systemHost = (Host) APILocator.getContentletAPI().find(systemHostId,systemUser,false);
+            //this.systemHost =  APILocator.getHostAPI().DBSearch(systemHostId, systemUser, false);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)